File: src/main/java/io/github/escapehonbab/jpa/objects/FriendChatInfo.java
package io.github.escapehonbab.jpa.objects;

import io.github.escapehonbab.jpa.model.BaseModel;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.Table;
import java.io.Serializable;

@Setter
@Getter
@Table(name = "chatroom")
@Entity
public class FriendChatInfo extends BaseModel implements Serializable {

    private byte[] img;
    private String name;
    private String userId;
    private String sex;
    private int age;
    private String time;
    private Long ownerId;

}

File: src/main/java/io/github/escapehonbab/jpa/objects/FriendChatInfo.kt
package io.github.escapehonbab.jpa.objects

import lombok.Builder
import java.io.Serializable
import javax.persistence.Entity
import javax.persistence.Table

@Builder
@Table(name = "chatroom")
@Entity
class FriendChatInfo(
    val img: ByteArray,
    val name: String,
    val userId: String,
    val sex: String,
    val age: Int,
    val time: String,
    val user: User
) : Serializable